### ISSUE TITLE
Add replay protection

### ISSUE THREAD

**Issue by MrHash** [btc2x/bitcoin#1]

**Comment by MrHash** [comment]
So far wipeout protection has been merged from https://github.com/jameshilliard/bitcoin/tree/segwit2x-wipeout-protection in  604bf781d2f32e5344c3c490abfa8644713b6653

**Comment by rodasmith** [comment]
This may not be needed yet. Could it be configured to kick in only when block size goes up to ~4 MB (8 MW)? That's the current idea at https://github.com/NYAcoin/bitcoin/issues/1

**Comment by MrHash** [comment]
Yes I think NYAcoin has the necessary development support so that maybe the best impl to endorse.

**Comment by rodasmith** [comment]
We are not yet ready to implement replay protection in NYAcoin because we first have to complete the analysis of the costs and benefits of a hard fork to 4 MB blocks. https://github.com/NYAcoin/bitcoin/issues/5 If it turns out that 2 MB blocks on a single chain is preferable, then our recommendation will likely be not to fork.

I invite both of the other independent teams of developers to issue any guidance on what details to cover in that review and to propose a process by which signatories could review and either ratify or reject the hard fork.
